src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.h

```diff
@@ -3,7 +3,8 @@
 
 #include "global.h"
 
-#define GET_KIBAKO2_COLLECTIBLE_ID(x) (((x->dyna.actor.params >> 0x8) & 0x7F) << 8)
+#define KIBAKO2_COLLECTIBLE_ID(x) (x->dyna.actor.params & 0x3F)
+#define KIBAKO2_COLLECTIBLE_FLAG(x) ((x->dyna.actor.params >> 0x8) & 0x7F)
 #define KIBAKO2_SKULLTULA_SPAWN_PARAM(x) (((x->dyna.actor.params & 0x1F) << 2) | 0xFF01)
 #define KIBAKO2_CONTENTS(x) ((x->dyna.actor.params >> 0xF) & 1)
```

Comment:
I think the only other thing I'd do here is `x -> this`, or maybe base it on `(thisx)->params` instead.

Reply:
`ObjKibako2_SpawnCollectible` takes a `this` instead of `thisx` right now, so I'll do the former

Reply:
Point with `thisx` is you can also pass `&this->actor` as the argument, but it's possible that wouldn't match anyway.

Reply:
Ah, I see. It does actually match with `(thisx)->params` but it definitely adds some bloat to pass `&this->dyna.actor` into every macro. Is `thisx` in macros preferred enough to justify this bloat?

Reply:
Hm, good question. I'd say they're most often used in Init, which is why `thisx` is a natural thing to think to use. But I defer to those who have looked at more actor files on which we'd prefer in general.

Reply:
Yeah it's a bit odd because `KIBAKO2_COLLECTIBLE_ID` and `KIBAKO2_COLLECTIBLE_FLAG` are in Init, where they could use `thisx` easily, *and* in `ObjKibako2_SpawnCollectible`, where they can't. `KIBAKO2_CONTENTS` is the same way. I'll leave this thread open for others to comment on, but for now I'll use `this`.

Reply:
Currently the standard is to use `thisx` and pass in the `Actor` pointer.

Reply:
Sounds good, I'll swap to `thisx` then.